src/style.css

```diff
@@ -2,2 +1,89 @@
+:root {
+  --main-color: #00acdc;
+}
+
+ul,
+ol {
+  padding: 0;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+ol,
+li,
+figure,
+figcaption,
+blockquote,
+dl,
+dd {
   margin: 0;
+  list-style: none;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For instance, 'Roboto', 'Arial', sans-serif could be specified as 'Roboto', 'Arial', 'Helvetica', sans-serif.